### DISCUSSION TITLE
Copilot not working at all on VS2022 ARM

### DISCUSSION THREAD

**Discussion by MichaelReaNZ** [community/community#30846] [Copilot Conversations]
Copilot has stopped working completely on VS2022 on Vs code it's fine.
Same issue with other versions of VS.

I'm paying for this I want it to work.

**Comment by iamcjwilliams** [comment]
@MichaelReaNZ We're sorry for the trouble here. Our Copilot team is working to improve the quality of the extension. Please visit this discussion to get the latest version: https://github.com/orgs/community/discussions/30716
